src/router/routes.js

```diff
@@ -341,6 +341,16 @@ module.exports = [
 		path: '/lending-home',
 		redirect: '/lend-by-category',
 	},
+	{
+		name: 'lenderProfile',
+		path: '/lender-vue/:publicId',
+		component: () => import('@/pages/LenderProfile/LenderProfile'),
+		meta: {
+			excludeFromStaticSitemap: true,
+			unbounceEmailCapture: true,
+			unbouncePopUp: true,
```

Comment:
Should these 2 unboune options be false. They are already signed up if they have a Kiva account.

Reply:
Following same case like Borrower Profile, this is a public page, like: https://www.development.kiva.org/lender/robert2236

Reply:
@mcstover Like Roger says, this is a public page that anyone can visit, not a logged-in only page.